App.jsx

```diff
@@ -1,43 +1,66 @@
-import React from "react";
-import { useSelector } from "react-redux";
-import { NavigationContainer } from "@react-navigation/native";
-import { createStackNavigator } from "@react-navigation/stack";
-import HomeScreen from "./components/HomeScreen";
-import SingleArticle from "./components/SingleArticle";
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { NavigationContainer } from '@react-navigation/native'
+import { createStackNavigator } from '@react-navigation/stack'
+import HomeScreen from './components/HomeScreen'
+import SingleArticle from './components/SingleArticle'
+import SignInScreen from './components/SignInScreen';
+import { FontAwesome5 } from '@expo/vector-icons'
 
-const Stack = createStackNavigator();
+const Stack = createStackNavigator()
 
 const App = () => {
-  const { appHeader } = useSelector(state => state)
+  const { appHeader } = useSelector((state) => state)
   return (
     <NavigationContainer>
       <Stack.Navigator>
         <Stack.Screen
           name="Meow"
           component={HomeScreen}
-          options={{
-            title: appHeader,
-            headerStyle: {
-              backgroundColor: "black",
-            },
-            headerTitleStyle: {
-              color: "rgb(240,230,140)",
-              fontSize: 25,
-              fontWeight: "bold",
+          options={(props) => ({
+            headerRight: () => {
+              return (
+                <FontAwesome5
+                  name="cat"
+                  size={24}
+                  color="white"
```

Comment:
would be nice to change the color of the logo when a user is logged in. but given the limited time, it is understandable if we don't change that.

Reply:
We have had that thought, but as a group we decided that time was not enough for this extra feature.